src/test/java/hudson/plugins/jira/JiraCreateIssueNotifierTest.java

```diff
@@ -168,7 +169,7 @@ public void performFailureSuccessIssueClosedWithComponents() throws Exception {
         doReturn(site).when(notifier).getSiteForProject(Mockito.any());
 
         Issue issue = mock(Issue.class);
-        Status status = new Status(null, null, JiraCreateIssueNotifier.finishedStatuses.Closed.toString() , null, null);
+        Status status = new Status(null, null, "", JiraCreateIssueNotifier.finishedStatuses.Closed.toString() , null, null);
```

Comment:
this is the reason of the test failure. Status name must cannot be empty in the test.

Reply:
I changed this to have a name which is not empty.